packages/rx_bloc_cli/bin/compile_bundles.sh

```diff
@@ -11,6 +11,11 @@ dart run mason_cli:mason bundle \
   -o lib/src/templates/ \
   mason_templates/bricks/rx_bloc_flavor_config
 
+dart run mason_cli:mason bundle \
```

Comment:
Please add a script for testing the newly introduced command similar to what we already have for the rx_bloc_cli create